### PR TITLE
Caching data every hour

### PR DESCRIPTION
**Pull request by christopherkade**
Added linter and prettier config

### CODE REVIEW

**Comment by Rude-Monkey** [comment]
:star: PR built and available in a preview environment **aya-experience-aya-day-weather-proxy-pr-4** [here](http://aya-day-weather-proxy.jx-aya-experience-aya-day-weather-proxy-pr-4.kube.gilleslabs.xyz)

**Comment by Rude-Monkey** [comment]
:star: PR built and available in a preview environment **aya-experience-aya-day-weather-proxy-pr-4** [here](http://aya-day-weather-proxy.jx-aya-experience-aya-day-weather-proxy-pr-4.kube.gilleslabs.xyz)

**Comment by Rude-Monkey** [comment]
:star: PR built and available in a preview environment **aya-experience-aya-day-weather-proxy-pr-4** [here](http://aya-day-weather-proxy.jx-aya-experience-aya-day-weather-proxy-pr-4.kube.gilleslabs.xyz)

**Review by Rude-Monkey — COMMENTED** [review]
Seems legit! What do you think @FBerthelot?

File: index.js [hunk 1]
```diff
@@ -29,33 +26,48 @@ router.get('/', async (ctx, next) => {
     { name: 'Montréal', lat: '45.50205700000001', lon: '-73.569345' },
   ];
 
+  const agencies = await Promise.all(
+    zenikaAgencies.map(zenikaAgencie =>
+      axios.get(
+        `https://api.darksky.net/forecast/${API_KEY}/${zenikaAgencie.lat},${zenikaAgencie.lon}`,
+      ),
+    ),
+  ).then(weatherDatas =>
```
Comment (FBerthelot) [inline index.js]: pourquoi un then, si tu pars en async await ?

**Review by FBerthelot — APPROVED** [review]

**Comment by Rude-Monkey** [comment]
:star: PR built and available in a preview environment **aya-experience-aya-day-weather-proxy-pr-4** [here](http://aya-day-weather-proxy.jx-aya-experience-aya-day-weather-proxy-pr-4.kube.gilleslabs.xyz)

**Pull request merged**